contrib/dist/linux/openmpi.spec

```diff
@@ -495,7 +495,7 @@ CXXFLAGS="%{?cxxflags:%{cxxflags}}%{!?cxxflags:$RPM_OPT_FLAGS}"
 FCFLAGS="%{?fcflags:%{fcflags}}%{!?fcflags:$RPM_OPT_FLAGS}"
 export CFLAGS CXXFLAGS FCFLAGS
 
-%configure %{configure_options} %{_configure_3rd_party}
```

Comment:
This seems like the wrong fix.  All you've done is make the configure line `./configure --with-<foo>=external --with-<foo>=<path>` and assume that configure will take the last option.  That's likely to always be true, but a better option seems to be handling this way earlier in the process, by being selective about how the `--with-<foo>=external` flags are added.

Reply:
In the update we inspect user provided `configure_options` and only set `--with-<pkg>=external` if:
- `all_external_3rd_party` is true
- user did not specify `--with-<pkg>=<path>` at all

I was looking for a better place to put this logic but landed here right before `configure ...`, since that is the only consumer of those options.